Fix idle harvesters (#1613)
Vinifera-Developers/Vinifera: docs/Bugfixes.md

- Fix a bug where a trigger's "Elapsed Time" event timers were reset when the trigger was already enabled and the "Enable Trigger" TAction was used on it.
- Fix a bug where the map would accept input while the user was in a dialog window.
- Fix a bug where the sidebar could only contain up to 75 items on a strip.

> - Fix a vanilla bug where harvesters will become permanently idle if they exhaust all resources to mine, even if new resources appears (e.g. tib tree)

Reviewer: ```suggestion
- Fix a vanilla bug where harvesters would become permanently idle if they exhausted all resources to mine, even if new resources appeared (e.g. spawned by a Tiberium tree)
```
